src/shared/lib/StoreUtils.ts

```diff
@@ -55,12 +57,21 @@ import {AlterationTypeConstants} from "../../pages/resultsView/ResultsViewPageSt
 import {stringListToIndexSet} from "public-lib/lib/StringUtils";
 import {GeneticTrackDatum_Data} from "../components/oncoprint/Oncoprint";
 import {normalizeMutations} from "../components/mutationMapper/MutationMapperUtils";
+import client from 'shared/api/cbioportalClientInstance';
 
 export const ONCOKB_DEFAULT: IOncoKbData = {
     uniqueSampleKeyToTumorType : {},
     indicatorMap : {}
 };
 
+export const CompleteProfileTypeSignature = {
```

Comment:
can you explain more what this is/how it relates to the data?

Reply:
are these now special values that will come back from the API?

Reply:
got it. is it possible to encode that in swagger?

Reply:
> also, could this be a ts Enum?

No, since the types that signify a whole genome approach can be string or _undefined_ a TS enum is not possible. I tried this :)

> got it. is it possible to encode that enum in swagger?

I do not understand this question. You mean whether the allowed variables could be made explicit in the swagger documentation of the backend?

Reply:
> You mean whether the allowed variables could be made explicit in the swagger documentation of the backend?

Yes, exactly

Reply:
I guess we can, but I do not know how. This is autogenerated from the Java code of the REST controllers isn't it? This feature is not yet implemented there. So, should I better remove this evaluation using unimplemented backend functionality? I am confused.

Reply:
oh I see. yeah, lets just do it in the frontend, and I think a string-valued enum as @alisman suggested would be perfect. And I think a different name would make it clearer.

```
export enum GenePanelIdSpecialValue {
    WHOLE_EXOME_SEQ="WES",
    ...
}
```

thoughts?

Reply:
@adamabeshouse 
> I think a string-valued enum as @alisman suggested would be perfect

No, since the types that signify a whole genome approach can be **string** or **undefined** a TS enum is not possible. I tried this :) The values must be of the same type...

Reply:
ahhh okay. then your approach sounds good. I still think the name should be changed, though

Reply:
Good suggestion. Variable name was updated.